pandas/tests/tools/test_to_timedelta.py

```diff
@@ -232,6 +233,7 @@ def test_to_timedelta_on_missing_values_list(self, val):
         actual = to_timedelta([val])
         assert actual[0]._value == np.timedelta64("NaT").astype("int64")
 
+    @pytest.mark.xfail(not IS64, reason="Floating point error")
```

Comment:
I think this is incorrect: I'm getting an XPASS here.

My guess is that you've hit this on i386, probably due to i387 weirdness. However, this wouldn't affect other 32-bit architectures or i386 builds with `-mfpmath=sse`. Gentoo is switching towards the latter since i387 is simply causing too many floating-point issues.

Reply:
A PR to make the xfail condition more precise would be welcome

Reply:
I know but I honestly have no clue how to detect the right case. At this point, I can think only of two options: either removing the xfail and ignoring i387-based x86 to support other 32-bit setups, or using an approximate comparison instead.